### PR TITLE
fix(curriculum): server timezone/date issue and format

### LINKED ISSUE #44475: Exercise Tracker - Issue with one test case due to timezones
## Affected page

[Exercise Tracker Project](https://www.freecodecamp.org/learn/back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker)

## Your code && Screenshots

You can check it deeply in the forum post: [Forum Post](https://forum.freecodecamp.org/t/exercise-tracker-datestring-format/488362)

## Expected behavior

To pass all test cases

## System

- Device: Laptop
- OS: Windows 10
- Browser: Edge
- Version: 96

## Additional context

One of the test case says that when the date field is empty it should take the actual date, so while the replit server is in one country and I'm in another when in my country might be Friday at night in the replit server will be Saturday at morning. And that's a problem because the test case error only shows that the format is no the right format and doesn't gives you any hint, this was possibly only with me being in front of the computer trying to realize why it doesn't work

### PR DESCRIPTION
**Pull request by kethangudi**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #44475 

Both submissions contain same code, in before case, submission is unfairly denied because the time zone of replit server differs from local time, causing incorrect insertion when checking for date. Possible to test yourself using this repl.it: https://boilerplate-project-exercisetracker-1.mickeymind.repl.co/

Before:  Test case uses local time zone which checks for date of April 07 (1 day ahead of submission date). 

      Replit uses UTC time zone which can be up to 14 hours behind local time.

image

After: Test case uses UTC time zone which checks for date of April 06 (actual submission date). 

      Put simply, both servers now only output and check for UTC time zone, so doesn't matter what time zone you submit from 
     (as long as the code works)

image

### CODE REVIEW

**Comment by mickeymind** [comment]
@farhanb99

**Review by thecodingaviator — APPROVED** [review]
This LGTM!

File: curriculum/challenges/english/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md [hunk 1]
```diff
@@ -519,7 +519,8 @@ async(getUserInput) => {
       description: 'test',
       duration: 60,
       _id,
-      date: new Date().toDateString()
+      date: new Date().toLocaleDateString("en-US", {timeZone: "UTC", weekday: "short", month: "short", 
+                                                    day: "2-digit", year: "numeric"}).replaceAll(',', '')
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md:523]:
```suggestion
      date: new Date().toLocaleDateString("en-US", {
        timeZone: "UTC", weekday: "short", month: "short",
        day: "2-digit", year: "numeric"
```

Nitpicking the formatting.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Clever solution! One nitpick on the formatting, then this looks good to me.

**Review by naomi-lgbt — APPROVED** [review]
okie dokie, looks good to me!

**Comment by lasjorg** [comment]
Does the requirement need to be changed to UTC?

---

Is the assumption here the server is Replit and that it is always UTC?

The test runs client-side (right?) against what comes back from a server. So right now, toDateString client/toDateString server. With this change, it is toUTCString client/toDateString server.

Maybe I'm just spacing out on the details, Date stuff is confusing.
